ivy/func_wrapper.py

```diff
@@ -393,7 +393,7 @@ def _handle_array_like_without_promotion(*args, **kwargs):
         ):
             annotation_str = str(annotation)
             if (
-                ("rray" in annotation_str or "Tensor" in annotation_str)
+                ("Array" in annotation_str or "Tensor" in annotation_str)
```

Comment:
this change would break support when the typehint is something like `np.ndarray` (the `a` in lowercase), which is why it's currently `rray in annotation_str`

Reply:
Ok, got it. Will revert this change.

Reply:
thanks!